### DISCUSSION TITLE
I just started using Towny...

### DISCUSSION THREAD

**Discussion by IrvTheNerve1** [TownyAdvanced/Towny#6661] [Q&A] [answered]
How do I configure Towny so that I don't need to use any of the money? Also how would someone usually go about setting Towny up alongside SiegeWar and whatnot? I am brand new at this and I want to give my server a good way to form their own little groups.

**Comment by LlmDl** [comment, marked as answer]
Towny's economy is disabled by setting using_economy: false in the towny config. 

SiegeWar has a installation guide, just like Towny.
